React/Modules/RCTDevMenu.h

```diff
@@ -58,6 +58,26 @@
 - (void)reload;
 
 /**
+ * Manually enable or disable remote JS debugging (can be called from JS)
```

Comment:
Wasn't sure if it was used for documentation purposes! Alright, I'll remove this.